app/controllers/registration_controller.rb

```diff
@@ -9,7 +9,7 @@ def create
     form_options
     insecure = params[:insecure] ? '--insecure' : ''
     args_query = "?#{registration_args.to_query}"
-    @command = "curl #{insecure} -X GET \"#{endpoint}#{args_query if args_query != '?'}\" #{headers} | bash"
+    @command = "curl #{insecure} -sX GET \"#{endpoint}#{args_query if args_query != '?'}\" #{headers} | bash"
```

Comment:
@stejskalleos `GET` is the default action so why is `-X GET` needed?

Reply:
I think I wanted to distinct it from other curls in template that are to same `/register` path but with `POST`.
But I guess users don't care about it much, so it can be removed.